s3store/s3store.go

```diff
@@ -536,3 +549,21 @@ func isAwsError(err error, code string) bool {
 	}
 	return false
 }
+
+func (store S3Store) CalcPartSize(size *int64) (effectivePartSize int64) {
```

Comment:
Efficiency. Mainly for my "testing the code for ALL possible file sizes from 1 Byte to 5 TB" experiment.